README.md

```diff
@@ -144,7 +144,7 @@ riffRaffUploadManifestBucket := Option("riffraff-builds")
 
 In order to upload artifacts and manifest files to the Guardian's RiffRaff buckets, you will need to give your build the appropriate permissions.
 
-1. Make sure your AWS account is in the [list of accounts](https://github.com/guardian/deploy-tools-platform/blob/master/cloudformation/riffraff-buckets.template.yaml) with permission to grant access to those buckets. If it is not, make a PR or ask @philwills to add it for you.
+1. Make sure your AWS account is in the [list of accounts](https://github.com/guardian/deploy-tools-platform/blob/main/cloudformation/riffraff-buckets.template.yaml) with permission to grant access to those buckets. If it is not, make a PR or ask @philwills to add it for you.
```

Comment:
Not related to this PR, but might as well...

```suggestion
1. Make sure your AWS account is in the [list of accounts](https://github.com/guardian/deploy-tools-platform/blob/main/cloudformation/riffraff-buckets.template.yaml) with permission to grant access to those buckets. If it is not, make a PR or ask @guardian/developer-experience to add it for you.
```

Reply:
Nice one, thanks!